src/test/java/seedu/heymatez/model/person/EmailTest.java

```diff
@@ -34,7 +34,9 @@ public void isValidEmail() {
         // missing parts
         assertFalse(Email.isValidEmail("@example.com")); // missing local part
         assertFalse(Email.isValidEmail("peterjackexample.com")); // missing '@' symbol
-        assertFalse(Email.isValidEmail("peterjack@")); // missing domain name
+        assertFalse(Email.isValidEmail("peterjack@.com")); // missing domain name
+        assertFalse(Email.isValidEmail("peterjack@examplecom")); // missing '.' symbol for top-level domain
```

Comment:
this is just a personal preference but i think that lines 38, 39 can be swapped so that the email "peterjack@example" can be above "peterjack@examplecom" so that it follows alphabetical ordering!

Reply:
Ok, noted